storage/innobase/os/os0file.cc

```diff
@@ -1256,8 +1256,10 @@ os_file_create_simple_func(
 
 #else /* __WIN__ */
 	int		create_flag;
+#ifdef WITH_INNODB_DISALLOW_WRITES
```

Comment:
Where from is WITH_INNODB_DISALLOW_WRITES? PS does not have this.

Reply:
It's from the wsrep source, it is a read-only like patch.